recipes/fpng/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get, save, load
+from conan.tools.build import check_min_cppstd
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class FpngConan(ConanFile):
+    name = "fpng"
+    description = "Super fast C++ .PNG writer/reader"
+    license = "unlicense",
```

Comment:
```suggestion
    license = "Unlicense",
```
Use the conventional capitalization for the SPDX identifier. Probably does not matter, though.